apps/supervisor/API_DOCUMENTATION.md

```
@@ -0,0 +1,707 @@
+└── .latest -> backup-2025-10-24T11-00-00/
+```
+
+---
+
+### Activity Reporter
+
+Reports workspace activity to the Dev8 Agent.
+
+**Configuration:**
+- `AGENT_ENABLED` - Enable/disable agent reporting (default: true)
+- `AGENT_URL` - Agent API endpoint (e.g., http://agent:8080)
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)